package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-cli",
-  "version": "2.3.2",
+  "version": "2.3.3",
```

Comment:
I would update to `2.4.0` instead of `2.3.3`, since we're [adding functionality in a backwards-compatible manner](http://semver.org/).